scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/RestResponseController.java

```diff
@@ -57,8 +63,14 @@
    * Given a known run name. get its metrics. If no run is found, null is returned. If there are
    * multiple runs with the same name that are from different sequencers, one is randomly selected.
    */
-  @GetMapping(value = "/run/{name}/metrics")
-  public HttpEntity<byte[]> getMetricsByName(@PathVariable("name") String id) {
+  @GetMapping("/run/{name}/metrics")
+  @ApiOperation(
+      value = "Get metrics by run name",
+      response = Json[].class,
+      responseContainer = "HttpEntity")
+  @ApiResponses({@ApiResponse(code = 200, message = "Success")})
```

Comment:
This endpoint responds with an empty array if the run doesn't exist, so the docs are correct in showing success only. It should be changed to return 404 status if the run isn't found though.